src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -53,6 +53,8 @@ public static class PropertyNames
 
             /// <summary>
             ///     Specifies the configuration property(ies) that provide the basic authentication credentials.
+            ///     USER_INFO: credentials are specified via the `schema.registry.basic.auth.user.info` config property in the form username:password.
+            ///     SASL_INHERIT: credentials are specified via the `sasl.username` and `sasl.password` configuration properties.
```

Comment:
Shouldn't this be an enum thing?

Reply:
it shouldn't be here - this string values would be best to match the JAVA config values. But it should be in the static config class, following the pattern of other configs, and the property should translate between the two names.

good catch.